sdk/communication/azure-communication-administration/README.md

```diff
@@ -289,7 +289,7 @@ Duration duration = Duration.ofSeconds(1);
 String phoneNumberSearchId = "SEARCH_ID_TO_PURCHASE";
```

Comment:
Could you also change variable names that reference the previous name of the model?